[INFO 2023-04-11 13:29:59,090]-> Logs will be saved to D:\Dev\THU-HyperG\examples\logs\2023-04-11--13-29-59\log.txt
[INFO 2023-04-11 13:29:59,091]-> Files in training will be saved in D:\Dev\THU-HyperG\examples\logs\2023-04-11--13-29-59
[INFO 2023-04-11 13:29:59,092]-> Random seed is 1681187399
[INFO 2023-04-11 13:29:59,094]-> A new study created in memory with name: no-name-baf4fdd3-b1d2-49af-ad2c-5e4f796293b8
[WARNING 2023-04-11 13:31:04,539]-> Trial 0 failed with parameters: {'k': 15, 'hidden_dim': 96, 'lr': 0.0009779409614487779, 'weight_decay': 0.006567211246082599} because of the following error: KeyboardInterrupt().
Traceback (most recent call last):
  File "C:\Users\Dutfo\AppData\Local\Programs\Python\Python39\lib\site-packages\optuna\study\_optimize.py", line 200, in _run_trial
    value_or_values = func(trial)
  File "d:\Dev\THU-HyperG\examples\dhg\experiments\hypergraphs\hypergraph.py", line 71, in experiment
    return super().experiment(trial)
  File "d:\Dev\THU-HyperG\examples\dhg\experiments\vertex_classification.py", line 83, in experiment
    return super().experiment(trial)
  File "d:\Dev\THU-HyperG\examples\dhg\experiments\base.py", line 103, in experiment
    val_res = self.validate(self.data, model)
  File "C:\Users\Dutfo\AppData\Local\Programs\Python\Python39\lib\site-packages\torch\autograd\grad_mode.py", line 27, in decorate_context
    return func(*args, **kwargs)
  File "d:\Dev\THU-HyperG\examples\dhg\experiments\hypergraphs\hypergraph.py", line 108, in validate
    return super().validate(data, model)
  File "C:\Users\Dutfo\AppData\Local\Programs\Python\Python39\lib\site-packages\torch\autograd\grad_mode.py", line 27, in decorate_context
    return func(*args, **kwargs)
  File "d:\Dev\THU-HyperG\examples\dhg\experiments\vertex_classification.py", line 130, in validate
    outputs = model(features, structure)
  File "C:\Users\Dutfo\AppData\Local\Programs\Python\Python39\lib\site-packages\torch\nn\modules\module.py", line 1130, in _call_impl
    return forward_call(*input, **kwargs)
  File "d:\Dev\THU-HyperG\examples\dhg\models\hypergraphs\hgnn.py", line 44, in forward
    X = layer(X, hg)
  File "C:\Users\Dutfo\AppData\Local\Programs\Python\Python39\lib\site-packages\torch\nn\modules\module.py", line 1130, in _call_impl
    return forward_call(*input, **kwargs)
  File "d:\Dev\THU-HyperG\examples\dhg\nn\convs\hypergraphs\hgnn_conv.py", line 57, in forward
    X = hg.smoothing_with_HGNN(X)
  File "d:\Dev\THU-HyperG\examples\dhg\structure\hypergraphs\hypergraph.py", line 1222, in smoothing_with_HGNN
    return L_HGNN.mm(X)
KeyboardInterrupt
[WARNING 2023-04-11 13:31:04,582]-> Trial 0 failed with value None.
